src/utils.jl

```diff
@@ -195,6 +195,15 @@ end
 @inline _gate(x::AbstractVector, h::Int, n::Int) = view(x, _gate(h, n))
 @inline _gate(x::AbstractMatrix, h::Int, n::Int) = view(x, _gate(h, n), :)
 
+@inline function init_hidden_state(rng::AbstractRNG, rnn, x::AbstractMatrix)
```

Comment:
Lets make these `_init_hidden_state` since we are not exposing them to the users